Form/ChangePassword.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace SumoCoders\FrameworkMultiUserBundle\Form;
+
+use Symfony\Component\Validator\Constraints as Assert;
+
+class ChangePassword
+{
+    /**
+     * @Assert\Length(
+     *     min = 6,
```

Comment:
imo moeten we application side niet te veel enforcen. Het lijkt me dan nuttiger om bij 6 karakters te tonen dat het een zwak wachtwoord is dan om mensen te verplichten om iets sterkers te kiezen (wat ze dan toch gaan vergeten of ergens opschrijven waar iedereen het kan vinden)